bindings/java/src/org/sleuthkit/datamodel/AbstractFile.java

```diff
@@ -1183,7 +1186,21 @@ public BlackboardArtifact newArtifact(int artifactTypeID) throws TskCoreExceptio
 		return getSleuthkitCase().newBlackboardArtifact(artifactTypeID, getId(), dataSourceObjectId);
 	}
 
-
+	@Override
```

Comment:
Why do you need this versus using the one in AbstractContent?

Reply:
We don't.  Removing this.